.all-contributorsrc

```diff
@@ -1,6 +1,6 @@
 {
   "projectName": "opensauced-guestbook",
-  "projectOwner": "TejasQ",
+  "projectOwner": "TeQ",
```

Comment:
We don't want to change this. Please change it back to the original.

Reply:
Hey, @adiati98. I don't know why this difference is showing up. When I'm on my main branch or my feature branch, I see the full 'TejasQ' on line 3. Can you advise on how I should locate the version of .all-contributorsrc that is giving this conflict?